builds/azure-pipelines/template-steps-build-test-ubuntu.yml

```diff
@@ -0,0 +1,69 @@
+parameters:
+  configuration: ''
+  solution: ''
+  nugetVersion: ''
+  testFilter: ''
+
+steps:
+- task: UseDotNet@2
+  displayName: 'Install .NET Core SDK 3.1.x for Build'
+  inputs:
+    version: '3.1.x'
+
+- script: npm install -g azure-functions-core-tools@3
+  displayName: 'Install Azure Functions Core Tools'
+
+- task: DockerInstaller@0
+  displayName: Docker Installer
+  inputs:
+    dockerVersion: 17.09.0-ce
+    releaseType: stable
+        
+- script: docker pull mcr.microsoft.com/mssql/server:2019-latest
+  displayName: Pull MSSQL Docker Image
+
+- bash: echo "##vso[task.setvariable variable=serverPassword]Test-$(Build.BuildNumber)"
```

Comment:
Since we're doing this already let's at least add in a timestamp or something to randomize it a bit more